zerver/tests/test_decorators.py

```diff
@@ -388,3 +479,38 @@ def test_webhook_deactivated_user(self):
         result = self.client.post(url, data,
                                   content_type="application/json")
         self.assert_json_error_contains(result, "Account not active", status_code=400)
+
+
+class TestValidateApiKey(AuthedTestCase):
+    def setUp(self):
+        self.webhook_bot = get_user_profile_by_email('webhook-bot@zulip.com')
+        self.default_bot = get_user_profile_by_email('hamlet@zulip.com')
```

Comment:
(It could make sense to add a new bot user with a name like default-bot@zulip.com for these tests)